examples/app.py

```diff
@@ -31,10 +31,10 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
-        pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
-        pip install invenio-assets==1.0.0a4
+        $ cdvirtualenv src/invenio-marc21
+        $ pip install -e .[all]
+        $ pip install "invenio-theme>=1.0.0a9"
+        $ pip install "invenio-assets>=1.0.0a4"
```

Comment:
IMHO examples are never going to be released on PyPI and should always work with current version. But of course I can add an upper limit. What do you propose - <=1.1 or <=2.0?